server/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "node index.js",
     "dev": "nodemon --exec npm start",
+    "build-map": "node generateLeafletMap.js",
```

Comment:
`Error: Cannot find module 'img2-leaflet-tiles'`
Your package is missing from the dependencies.

Reply:
Oh forgot to re-include the library in package.json